src/tests/EdFi.DataManagementService.Tests.E2E/StepDefinitions/StepDefinitions.cs

```diff
@@ -222,9 +222,17 @@ public async Task WhenAPUTRequestIsMadeToReferencedResourceWith(string url, stri
             _apiResponse = await _playwrightContext.ApiRequestContext?.PutAsync(url, new() { Data = body })!;
             if (_apiResponse.Status != 204)
             {
-                JsonNode responseJson = JsonNode.Parse(_apiResponse.TextAsync().Result)!;
+                var result = _apiResponse.TextAsync().Result;
+                _logger.log.Information(result);
 
-                _logger.log.Information(responseJson.ToString());
```

Comment:
Previously, if the response wasn't JSON, then we would get an exception before the logger was used.